src/api/actions-spec.ts

```diff
@@ -54,3 +54,8 @@ export interface ActionsSpecPostRequestBody
 // Almost no changes, omitting old `redirect`
 export interface ActionsSpecPostResponse
   extends Omit<SolanaPaySpecPostResponse, 'redirect'> {}
+
+export interface Options {
+  // For multi-chain, see https://github.com/ChainAgnostic/CAIPs
+  blockchainIds?: string;
```

Comment:
I understand we didn't want to support multiple ids yet, but maybe we should use `string[]` type to avoid breaking change in future?